src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -1217,7 +1231,10 @@ REDHAWK_PALEXPORT bool PalGetMaximumStackBounds(_Out_ void** ppStackLowOut, _Out
 REDHAWK_PALEXPORT Int32 PalGetModuleFileName(_Out_ const TCHAR** pModuleNameOut, HANDLE moduleBase)
 {
     Dl_info dl;
+    // Emscripten's implementation of dladdr corrupts memory and doesn't have the real name, so skip it
```

Comment:
I have the same concern as for `PalGetModuleHandleFromPointer`.

Reply:
This is only used for user-facing things like diagnostics. Replaced with a hard-coded string that should be easy to find if we want to revisit it.